sphinx/source/bokeh/static/custom.css

```diff
@@ -206,15 +206,15 @@
     color: #542437;
   }
 
-/* reduce margin and height from 80px and -80px, https://github.com/bokeh/bokeh/issues/9915 */
+/* Assigned h[n]::before styling to role main to account for both content and footer headers, https://github.com/bokeh/bokeh/issues/9915 */
 /* offsetting html anchor titles to adjust for fixed header, https://github.com/pandas-dev/pandas-sphinx-theme/issues/6*/
-  h2::before, h3::before,
+[role=main]h2::before, h3::before,
```

Comment:
Two issues:
1. Just `main` should be used instead of `[role=main]` in this case. See the "Description" section of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role
2. It has to be added to each part - `main h2::before, main h3::before, ...`. Otherwise, it only affects the `h2::before` selector.

Reply:
That makes complete sense, I think everything should be correct now